.github/workflows/scripts/css_check.py

```
@@ -105,6 +105,99 @@ def check_embedded_styles(
+            match_start: The starting position of the match
+            match_end: The ending position of the match
+            
+        Returns:
+            bool: True if it's a MUI component prop, False otherwise
+        """
+        # Look at what comes after the match (the part after =)
+        after_match = code_line[match_end:].lstrip()
+        
+        # MUI component props use = with a quoted string value
+        # e.g., fontSize="small" or fontSize='small'
+        if after_match.startswith('"') or after_match.startswith("'"):
```

Ruff (0.14.10)

Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)